mozilla/fx-private-relay
# Add tracker removal UI

# New feature description

This adds UI to enable/disable tracker removal in the settings page. It will only show up when the back-end is updated to support the setting.

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/173029794-a347481e-c9e8-4758-a4a2-19457140ba3e.png)

![image](https://user-images.githubusercontent.com/4251/173029815-91446b47-824f-4ffa-a2fe-f6752a9e2231.png)

# How to test

You can currently only view this in the demo site, where the API mocks have been updated to mock the required back-end changes. Simply go to the settings page, where you should see the new setting.

# Checklist

- [x] l10n dependencies have been submitted: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/91
